kubernetes/reporter/src/main/java/org/arquillian/cube/kubernetes/reporter/KubernetesSection.java

```diff
@@ -0,0 +1,23 @@
+package org.arquillian.cube.kubernetes.reporter;
+
+
+import org.arquillian.reporter.api.event.SectionEvent;
+import org.arquillian.reporter.api.event.TestSuiteSection;
+import org.arquillian.reporter.api.model.report.BasicReport;
+
+public class KubernetesSection extends SectionEvent<KubernetesSection, BasicReport, TestSuiteSection>{
+
+    public KubernetesSection() {
+        super("k8s");
+    }
+
+    @Override
+    public TestSuiteSection getParentSectionThisSectionBelongsTo() {
+        return new TestSuiteSection() ;
```

Comment:
Same as I've commented in DockerLogSection - missing possibility to specify test suite id